api/v1alpha1/alluxioruntime_types.go

```diff
@@ -76,6 +76,18 @@ type AlluxioCompTemplateSpec struct {
 	// NodeSelector is a selector which must be true for the master to fit on a node
 	// +optional
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
+
+	// Journal is a journal setting for embedded raft journal
+	Journal Journal `json:"journal,omitempty"`
```

Comment:
Can this be transparent to the end user?

Reply:
I think there is no need to be transparent for user, it will make it more complicate.

Reply:
I think we have to discuss with this.